internal/eliza/eliza.go

```diff
@@ -36,6 +36,18 @@ func Reply(input string) (string, bool) {
 	return lookupResponse(input), false
 }
 
+// GetIntroResponses returns a collection of introductory responses tailored to the given name.
+func GetIntroResponses(name string) []string {
+	intros := []string{}
```

Comment:
nit: perf here doesn't matter, but we might as well do `intros := make([]string, 0, len(introResponses)+2)`